### ISSUE TITLE
Feature request: JDB support

### ISSUE THREAD

**Issue by noboruma** [sakhnik/nvim-gdb#66]
Is there any plan to support JDB?
Might be trickier than gdb.. e.g.: I have not found any command to retrieve every breakpoints that are being set.
That said, the other commands/workflow should be similar?

**Comment by sakhnik** [comment]
I certainly don't have such plans. But I believe you definitely could extend JVM in a transparent manner to implement missing commands. Just like we do for LLDB via their Python API and GDB using PTY. Please feel free to offer a PR if you dare ;-)

**Comment by sakhnik** [comment]
Well, Java is designed for IDE!

**Comment by noboruma** [comment]
Hum I have to disagree with that statement ;-)
I tried to implement it myself, but I found it hard to follow. If there is any wiki page that would explain what need to be implemented for your plugin to work, that would be helpful!
I ended up using vebugger plugin for now.

**Comment by sakhnik** [comment]
Yes, unfortunately, no wiki or anything. But I've just added support for bashdb starting from the commit 14842fb8e47375440c2f5c2dd3e659dc2a169165. Basically, one needs to create a state machine under `backends` and a proxy app to query breakpoints in the background. The state machine would parse debugger's output to determine what part of source code to open and position the cursor at. If the debugger can't list set breakpoints, you're out of luck to displaying them. That's it.
